androidHyperskillApp/src/main/res/layout/layout_next_learning_activity.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    >
+
+    <TextView
+        android:id="@+id/homeNextLearningActivityTitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/home_next_learning_activity_title"
+        android:textAppearance="@style/TextAppearance.AppCompat.Body1"
+        android:textColor="@color/color_on_surface_alpha_60"
+        android:layout_marginHorizontal="20dp"
+        />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/homeNextLearningActivityRecycler"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```

Comment:
Why do we use here `match_parent` instead of `wrap_content`?

Reply:
Because recyclerView is not a regular view: it is a special lazy view, which reuses views for performant drawing lists.
Setting `wrap_content` break the lazy nature of the recyclerView thus that it will create as many views as the list contains.
In this exact example, it is possible to use `wrap_content`, because we always have only 1 item in the list. However, the recycler view still wraps the content even with the `match_parent` height.